src/schema/babelon.yaml

```diff
@@ -10,6 +10,9 @@ prefixes:
   rdfs: http://www.w3.org/2000/01/rdf-schema#
   rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
   oboInOwl: http://www.geneontology.org/formats/oboInOwl#
+  HP: http://purl.obolibrary.org/obo/HP_
```

Comment:
This should not be necessary, but it does not harm. See my last question in the linkml channel.